Makefile

```diff
@@ -1,8 +1,13 @@
+.PHONY: hashing json
```

Comment:
What does this line do?

Reply:
see: https://stackoverflow.com/a/2145605

> These special targets are called phony and you can explicitly tell Make they're not associated with files.

Reply:
Ah cool :) Learn something every day!